inst/include/RcppSimdJson/deserialize/Type_Doctor.hpp

```diff
@@ -59,7 +59,7 @@ class Type_Doctor {
 
 template <Type_Policy type_policy, utils::Int64_R_Type int64_opt>
 inline Type_Doctor<type_policy, int64_opt>::Type_Doctor(simdjson::dom::array array) noexcept {
```

Comment:
This method is marked `noexcept`. In the PR, I do "`int64_t(element)`" but this throw if element is not `int64_t`. If there is ever a bug somewhere, there won't be an exception thrown, I think, but instead you will get an abort of some kind.

Why is it marked noexcept? Is there a reason?

We can rewrite it in a no-exception form but then we have to handle the error condition somehow... like...
```C++
if(int64_t value; element.get(value) == simdjson::SUCCESS) {
  // use value
} else {
  // fall back?
}

Reply:
It's @knapply 's code so I am half guessing but likely the reason we do not do `int64_t` assignment is ... that R does have native 64 bit integeers which is we need 'doctoring' to go to double, or smaller int, or string, or ... and have all these options to cope with `int64_t` payloads.